Configuration/webapp/app/test/public/page-root-mocha.cjs

```diff
@@ -0,0 +1,55 @@
+const assert = require('assert');
+// const test = require('../mocha-index.cjs');
```

Comment:
There should not be commented code in the codebase. 
Either is needed, case in which we do not comment it, or is not needed, case in which we remove